code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -20,8 +20,12 @@ LINEN BINS
 	item_color = "white"
 	resistance_flags = FLAMMABLE
 	slot_flags = SLOT_FLAG_BACK
-
 	dog_fashion = /datum/dog_fashion/head/ghost
+
+	/// The associated key to the dye registry GLOB list, used to transform/color this piece of clothing through dyeing
+	var/dyeing_key = ""
```

Comment:
This isn't really ever used as a string in code, is it? We should specify its defines instead. Using the autodoc comment above
```suggestion
	var/dyeing_key
```

Reply:
Fair point, I should probably move the dyeing key var to the item level anyway for simplicity sake

Reply:
fixed in next commit